ampd/src/solana/json_rpc.rs

```diff
@@ -0,0 +1,32 @@
+use async_trait::async_trait;
+use ethers::providers::{JsonRpcClient, ProviderError};
+use mockall::automock;
+use solana_transaction_status::EncodedConfirmedTransactionWithStatusMeta;
+
+use crate::json_rpc::Client;
+
+type Result<T> = error_stack::Result<T, ProviderError>;
+
+#[automock]
+#[async_trait]
+pub trait SolanaClient {
+    // async fn get_transactions(&self, digests: Vec<Transaction>) -> Result<Vec<Transaction>>;
```

Comment:
We can remove that if not needed.

Reply:
fixed at 484e3a4c30157696679548a518f58a3d5cd83c27